client/js/components/procedure/admin/DpAddOrganisationList.vue

```diff
@@ -322,14 +638,117 @@ export default {
       }, [])
     },
 
+    setAppliedFilterQuery (filter) {
+      // Remove groups from filter
+      const selectedFilterOptions = Object.fromEntries(Object.entries(filter).filter(([_key, value]) => value.condition))
+      const isReset = Object.keys(selectedFilterOptions).length === 0
+      const isAppliedFilterQueryEmpty = Object.keys(this.appliedFilterQuery).length === 0
+
+      if (!isReset && isAppliedFilterQueryEmpty) {
+        Object.values(selectedFilterOptions).forEach(option => {
+          this.appliedFilterQuery[option.condition.value] = option
+        })
+      } else if (isReset) {
+        const filtersWithConditions = Object.fromEntries(
+          Object.entries(this.filterQuery).filter(([key, value]) => value.condition)
+        )
+
+        this.appliedFilterQuery = Object.keys(filtersWithConditions).length ? filtersWithConditions : {}
+      } else {
+        this.appliedFilterQuery = selectedFilterOptions
+      }
```

Comment:
Just a Suggestion. This should be the same with two less conditions:
```suggestion
      this.appliedFilterQuery = selectedFilterOptions

      if (isReset) {
        const filtersWithConditions = Object.fromEntries(
          Object.entries(this.filterQuery).filter(([key, value]) => value.condition)
        )

        this.appliedFilterQuery = Object.keys(filtersWithConditions).length ? filtersWithConditions : {}
      } else if (isAppliedFilterQueryEmpty) {
        Object.values(selectedFilterOptions).forEach(option => {
          this.appliedFilterQuery[option.condition.value] = option
        })
      }
```
(not tested)